db/schema.rb

```diff
@@ -37,7 +37,7 @@
   end
 
   create_table "categories", force: :cascade do |t|
-    t.text "name"
+    t.string "name"
```

Comment:
It's a good refactoring, but out of scope. Keep this kind of edits out from this PR. It's about showing the profile page.

Reply:
Not really refactoring, just ran `db:migrate`, and this means it wasn't run in the last merged PR

Reply:
Okay